src/pages/set-user-info/components/SetUserGender.tsx

```
@@ -0,0 +1,53 @@
+        ...prev,
+        gender: value,
+      };
+    });
+  };
+
+  return (
+    <div className='flex flex-1 flex-col gap-[1rem] py-[2rem]'>
+      <h2 className='title-sb-16 px-[0.5rem]'>{title}</h2>
+      <div className='flex w-full gap-[1rem]'>
+        <Button
+          children={male}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)